package.json

```diff
@@ -107,6 +109,7 @@
     "@angular/ssr": "19.0.5",
     "@netlify/functions": "^3.0.0",
     "@nx/devkit": "20.3.0",
+    "@phenomnomnominal/tsquery": "^6.1.3",
```

Comment:
As 3rd party deps in prod code is a think to think of, did you compare package health stats with `https://www.npmjs.com/package/ts-morph?activeTab=versions`or other libs? Just a thought.